jumpscale/clients/explorer/explorer.py

```diff
@@ -15,11 +15,9 @@
 class Explorer(Client):
     url = fields.String()
 
-    def __init__(self, **kwargs):
-
+    def __init__(self, url, **kwargs):
```

Comment:
I think it's better if the user knows about this when he calls get/new of the factory.

See how `new` of stellar is overridden [here](https://github.com/js-next/js-ng/blob/development/jumpscale/clients/stellar/__init__.py) to add `secret` parameter.

As @grimpy said before, he does not like undocumented `kwargs`.